Misc/NEWS.d/next/Documentation/2020-12-17-14-23-16.bpo-42669.fjuDXY.rst

```diff
@@ -0,0 +1 @@
+Explain that :keyword:`except` no longer accepts nested tuples.  (This was changed in Python 3.0, but not previously documented.)
```

Comment:
I don't think we need the blurb for this change. It's not substantive enough to be in the News file.